modules/nested-tabs-html/assets/scss/frontend.scss

```diff
@@ -1,6 +1,351 @@
+	--n-tabs-title-align-items-toggle: center;
+	--n-tabs-title-justify-content: center;
+	--n-tabs-title-align-items: center;
+	--n-tabs-title-direction: row;
+	--n-tabs-title-gap: 10px;
+	--n-tabs-title-flex-grow: 0;
+	--n-tabs-title-flex-basis: content;
+	--n-tabs-title-flex-shrink: initial;
+	--n-tabs-title-order: initial;
+	--n-tabs-title-padding-top: var(--n-tabs-default-padding-block);
+	--n-tabs-title-padding-right: var(--n-tabs-default-padding-inline);
+	--n-tabs-title-padding-bottom: var(--n-tabs-default-padding-block);
+	--n-tabs-title-padding-left: var(--n-tabs-default-padding-inline);
+	--n-tabs-title-border-radius: initial;
+	--n-tabs-title-transition: 0.3s;
+	--n-tabs-icon-color: var(--e-global-color-secondary, var(--n-tabs-color-secondary-fallback));
+	--n-tabs-icon-color-hover: var(--n-tabs-title-color-hover);
+	--n-tabs-icon-color-active: #{$white};
+	--n-tabs-icon-gap: 5px;
+	width: 100%;
+	max-width: 100%; /* Fix issue with new created n-tabs inside n-tabs with overflow */
+
+	.e-n-tabs {
+		display: var(--n-tabs-display);
+		flex-direction: var(--n-tabs-direction);
+		gap: var(--n-tabs-gap);
+		text-align: $start;
+		min-width: 0;
+
+		&-heading {
+			display: var(--n-tabs-heading-display);
+			flex-basis: var(--n-tabs-heading-width);
+			flex-direction: var(--n-tabs-heading-direction);
+			flex-shrink: 0;
+			justify-content: var(--n-tabs-heading-justify-content);
+			gap: var(--n-tabs-title-gap);
+			overflow-x: var(--n-tabs-heading-overflow-x);
+			flex-wrap: var(--n-tabs-heading-wrap);
+			-ms-overflow-style: none; /* IE and Edge */
+			scrollbar-width: none; /* Firefox */
+
+			&::-webkit-scrollbar {
+				display: none; /* Hide scrollbar for Chrome, Safari and Opera */
+			}
+
+			&.e-scroll {
+				cursor: grabbing;
+				cursor: -webkit-grabbing;
+
+				&-active {
+					position: relative;
+
+					&::before {
+						content: '';
+						position: absolute;
+						inset-block: 0;
+						inset-inline: -1000vw;
+						z-index: $second-layer;
+					}
+				}
+			}
+		}
+
+		&-content {
+			display: var(--n-tabs-content-display);
+			flex-grow: 1;
+			min-width: 0;
+
+			> .e-con:not(.e-active) {
+				display: none;
+			}
+		}
+	}
+
+	.e-n-tab {
+
+		&-title {
+			all: unset;
+			display: flex;
+			align-items: var(--n-tabs-title-align-items-toggle, var(--n-tabs-title-align-items));
+			flex-direction: var(--n-tabs-title-direction);
+			justify-content: var(--n-tabs-title-justify-content-toggle, var(--n-tabs-title-justify-content));
+			gap: var(--n-tabs-icon-gap);
+			border-width: var(--n-tabs-border-width);
+			position: relative;
+			cursor: pointer;
+			outline: none;
+			flex-grow: var(--n-tabs-title-flex-grow);
+			flex-basis: var(--n-tabs-title-flex-basis);
+			flex-shrink: var(--n-tabs-title-flex-shrink);
+			padding-top: var(--n-tabs-title-padding-top);
+			padding-right: var(--n-tabs-title-padding-right);
+			padding-bottom: var(--n-tabs-title-padding-bottom);
+			padding-left: var(--n-tabs-title-padding-left);
```

Comment:
use logical properties from properties, values and variables.

Reply:
Updated